### PR TITLE
Add crypto.randomBytes

### PR DESCRIPTION
**Pull request by bookmoons**
Adds method `randomBytes` to `k6/crypto`. The method takes a single parameter `size` specifying byte count and returns an array of random bytes. Bytes are represented as number literals.

Toward #900.

### CODE REVIEW

**Comment by bookmoons** [comment]
AppVeyor failed on this, but it seems to be from a different part of the code. I don't think it's from the changes:

```
--- FAIL: TestRealTimeAndSetupTeardownMetrics (5.97s)
```

**Comment by bookmoons** [comment]
Codecov is showing 95.72%, but running locally I get 96.1%. Is it possible it doesn't rerun if the CI builds fail?

**Comment by MStoykov** [comment]
LGTM!
That test is known to be problematic #919 and I am thinking of disabiling it for now on appveyor as it mostly fails for no reason. On the point of coverage - I think codecov doesn't update when only files it doesn't track are changed and in this case only test are changed ... which is also not ideal :(

File: js/modules/k6/crypto/crypto.go [hunk 1]
```diff
@@ -50,6 +51,15 @@ func New() *Crypto {
 	return &Crypto{}
 }
 
+func (*Crypto) RandomBytes(ctx context.Context, size int) []byte {
+	bytes := make([]byte, size)
```
[the remark this answers is not in the document]
Comment (bookmoons) [reply js/modules/k6/crypto/crypto.go]: Thanks @na--. Updated that.

I'm not sure of a good way now to hit the second error branch in a test. Does that deserve more attention?

[the remark this answers is not in the document]
Comment (MStoykov) [reply js/modules/k6/crypto/crypto.go]: Given that crypto.Reader is a global var you can replace it with an implementation that returns an error on every call ;) Obliviously you will need to return the original reader and not do it in a Parallel test

[the remark this answers is not in the document]
Comment (bookmoons) [reply js/modules/k6/crypto/crypto.go]: OK, tried it that way. A local run now gives coverage of 97.5%.

**Comment by bookmoons** [comment]
Should I be updating the copyright date on files I modify? It's a couple years old now.

**Comment by MStoykov** [comment]
I am pretty sure we should make some script or use some tool to update this automatically ...

**Comment by bookmoons** [comment]
Thanks for merging guys.

**Pull request merged**